### ISSUE TITLE
Model importing problem

### ISSUE THREAD

**Issue by 44vladimirov** [openvinotoolkit/openvino#12445]
Good day!

I am trying to import previously exported model using [OpenVINO 2022.1](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/) with the following simple code.

```C++
#ifdef API1
#include <ie/inference_engine.hpp>
#else
#include <openvino/openvino.hpp>
#endif

#include <fstream>

int main(int const argc, char const* argv[]) {
	std::string modelPath = "model.xml";
	std::string device = "CPU";
	std::string exportPath;

	if (argc > 1) {
		modelPath = argv[1];
	}
	exportPath = modelPath + ".blob";

	try {
#ifdef API1
		InferenceEngine::Core core;
		auto network = core.ReadNetwork(modelPath);
		auto execNetwork = core.LoadNetwork(network, device);
		execNetwork.Export(exportPath);
		execNetwork = core.ImportNetwork(exportPath, device);
		auto request = execNetwork.CreateInferRequest();
#else
		ov::Core core;
		auto model = core.read_model(modelPath);
		auto compiled_model = core.compile_model(model, device);

		std::ofstream writeStream(exportPath, std::ios::binary);
		compiled_model.export_model(writeStream);
		writeStream.close();

		std::ifstream readStream(exportPath, std::ios::binary);
		compiled_model = core.import_model(readStream, device);
		readStream.close();

		auto request = compiled_model.create_infer_request();
#endif
	}

	catch (std::exception const& err) {
		std::cerr << err.what() << std::endl;
		return -1;
	}

	return 0;
}
```

With the model [vehicle-detection](https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/3/vehicle-detection-0202/FP32/) the program runs fine and exits successfully.

But with the model [poconetlike](https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/3/noise-suppression-poconetlike-0001/FP32/) it fails during `core.import_model(readStream, device)` or `core.ImportNetwork(exportPath, device)` with the same exception `ir_deserializer.cpp:715 Cannot create Subgraph layer Multiply_289158 id:68 from unsupported opset: SnippetsOpset`.

Is it a supposed behavior?

**Comment by ilya-lavrenov** [comment]
CC @dmitry-gorokhov @a-sidorova

**Comment by ilya-lavrenov** [comment]
I suppose this PR needs to be merged to fix the issue

**Comment by andrei-kochin** [comment]
@44vladimirov is the issue is still visible on latest master after merging of #14064 ?

**Comment by ilya-lavrenov** [comment]
Please, reopen if the issue is still here.
Fixed version is 2022.3
